Loop/Window Management/Window.swift

```diff
@@ -177,10 +232,39 @@ class Window {
         fullscreen = !fullscreen
     }
 
+    /// Check with the `NSRunningApplication` if the app is hidden (⌘H).
     var isHidden: Bool {
```

Comment:
```suggestion
    var isApplicationHidden: Bool {
```
This will help distinguish it from `isWindowHidden`.